core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
Honestly, at this point, I have no idea. Can someone describe what exactly we are trying to achieve with this PR? Is it useful in the long run?

It seems to me to make more sense to build the job storage layer first, and then write this part to add jobs to it.